src/simple_cache_client.rs

```diff
@@ -96,11 +96,28 @@ impl SimpleCacheClientBuilder {
     ///         .build();
     /// # })
     /// ```
+    // TODO: Add test with token and endpoint
     pub fn new(auth_token: String, default_ttl_seconds: NonZeroU64) -> Result<Self, MomentoError> {
         SimpleCacheClientBuilder::new_with_explicit_agent_name(
             auth_token,
             default_ttl_seconds,
             "sdk",
+            None,
+        )
+    }
+
+    /// Like new() above, but requires a momento_endpoint.
+    // TODO: Update the documentation and deprecate the existing new method.
```

Comment:
wonder if it would be worth adding a `once xyz happens` to this TODO so people reading this know when we plan to actually get to it?